travis/test_server.py

```diff
@@ -312,6 +337,11 @@ def main(argv=None):
     test_enable = str2bool(os.environ.get('TEST_ENABLE', True))
     dbtemplate = os.environ.get('MQT_TEMPLATE_DB', 'openerp_template')
     database = os.environ.get('MQT_TEST_DB', 'openerp_test')
+    is_runbot = str2bool(os.environ.get('RUNBOT'))
+    is_gitlab_ci = str2bool(os.environ.get('GITLAB_CI'))
+    is_travis_ci = str2bool(os.environ.get('TRAVIS'))
```

Comment:
Why do you consider travis CI?

Remember Travis CI manages its own PostgreSQL settings which may not be customized AFAIK.

Reply:
I'm not patching PostgreSQL.
I'm patching odoo to create "unlogged" tables and travis kill the build after success then IMHO It is a good site to use "unlogged" too.

Reply:
and we can see that it is working.
https://travis-ci.org/OCA/maintainer-quality-tools/jobs/427705355#L831-L833